ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -413,6 +505,22 @@ const NetworksForm = ({
         }
       }
 
+      if (
+        addNewNetwork &&
+        Object.values(orderedNetworksList).some(
```

Comment:
I think storing the Object.values(...) result in a named var would be good for legibility & future maintenance.